fili-core/src/test/java/com/yahoo/bard/webservice/models/druid/client/impl/TestDruidWebService.java

```diff
@@ -111,7 +113,7 @@ public void postDruidQuery(
         // Invoke failure callback if we have a throwable to give it
         if (throwable != null) {
             failure.invoke(throwable);
-            return;
+            return ConcurrentUtils.constantFuture(null);
```

Comment:
This is actually changed, somehow github didn't catch it. [Link](https://github.com/yahoo/fili/pull/254/commits/d4054a96c200ed06ea7121ecc355b200b50114dc#diff-dcbc81d29aa4af827abee6b6fe2819e1R118)